packages/opentelemetry-tracing/src/BasicTracerProvider.ts

```diff
@@ -31,6 +32,7 @@ export class BasicTracerProvider implements types.TracerProvider {
 
   activeSpanProcessor = new NoopSpanProcessor();
   readonly logger: types.Logger;
+  readonly resource: Resource = Resource.createTelemetrySDKResource();
 
   constructor(private _config: TracerConfig = DEFAULT_CONFIG) {
```

Comment:
Same here, otherwise no option for users to configure the resource with `TracerProvider`.